game_event/test_game_event_views.py

```diff
@@ -0,0 +1,66 @@
+import pytest
+from django.urls import reverse
+from .models import GameEvent, Court
+from decimal import Decimal
+from .views import game_events
+from django.utils import timezone
+
+
+@pytest.mark.django_db
+class TestGameEventServerResponses:
+    @pytest.fixture
+    def saved_court(self):
+        court = Court.objects.create(x=Decimal('11'), y=Decimal('22'),
+                                     city='TEST_CITY', neighborhood='TEST_NEIGHBORHOOD', max_players=30)
+        return court
+
+    @pytest.fixture
+    def saved_game_event(self, saved_court):
+        game_event = GameEvent.objects.create(
+            id=1000,
+            time=timezone.now() - timezone.timedelta(days=1),
+            level_of_game=5,
+            min_number_of_players=4,
+            max_number_of_players=10,
+            court=saved_court,
+            ball_game='Basketball'
+        )
+        return game_event
+
+    def test_success_loading_site(self, client):
```

Comment:
You have a bunch of other tests that successfully load the page so I think what you need to test here is that you can load the page with no filters.
```suggestion
    def test_loading_site_no_filters(self, client):
```